tests/e2e/vsan_stretched_cluster.go

```diff
@@ -3374,14 +3512,14 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 
 		for i := 0; i < 6; i++ {
 			framework.Logf("Creating pvc %v with reclaim policy Delete", i)
```

Comment:
I think we can create a seperate util for creating multiple pods and use that same util in all testcases instead of repeating same code again and again